reference/actions/log.md

```diff
@@ -50,6 +51,18 @@ The merged custom log fields are:
 }
 ```
 
+They would appear in the [access log](/administration/logging.md#access-log) like this:
+
+```json
+{
+  "timestamp": "2019-10-15T15:49:13+00:00",
+  "type": "flat_access",
+  …
+  "name": "alice",
```

Comment:
`name` and `role` should appear in the `user` property, shouldn't they?

Reply:
Hawkeye…